README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
In our internal project, we are having "TestServer" class which is a simple jetty instance that is having testing servlet that returns proper messages and response codes. It works for simple cases and is fast enough.